pepc.js

```diff
@@ -0,0 +1,53 @@
+const API_ACCESS_STATUSES = {
+    success: '🟢 success',
+    error: '🔴 error',
+    unknown: '⚫️ unknown',
+};
+// Display the feature access status (whether the feature can actually be accessed successfully in the browser)
+function updateAccessStatus(permissionName, accessStatus, message) {
+    const textToDisplay = API_ACCESS_STATUSES[accessStatus];
+    document.querySelector(`#${permissionName}-access-status`).innerText =
+        textToDisplay;
+    if (message) {
+        document.querySelector(`#${permissionName}-error-message`).innerText = message
+    }
+}
+// Utils
```

Comment:
Ideally we'd share the code, but since we're not using ESM I'd at least reference the source.
```suggestion

// Utils (copied from `index.js`)
```